examples/torch/pearl_half_cheetah_vel.py

```diff
@@ -0,0 +1,151 @@
+import akro
+import numpy as np
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.envs.env_spec import EnvSpec
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
+from garage.experiment import LocalRunner, run_experiment
+from garage.sampler import PEARLSampler
+from garage.torch.algos import PEARLSAC
+from garage.torch.embeddings import RecurrentEncoder
+from garage.torch.modules import MLPEncoder
+from garage.torch.q_functions import ContinuousMLPQFunction
+from garage.torch.policies import ContextConditionedPolicy, \
+    TanhGaussianMLPPolicy
+import garage.torch.utils as tu
+
+params = dict(
```

Comment:
i can see why this style is useful for experiments, but for examples we usually place all these params inline. if some of them are sane defaults, it's okay to make them the default argument in the algorithm/primitive and omit them from the example.